draft-iab-privacy-partitioning.md

```diff
@@ -81,6 +81,14 @@ is no link between user-specific identity and user-specific data. However, appli
 privacy partitioning helps ensure that user privacy violations becomes more technically difficult
 to achieve over time.
 
+RFC6973 already discusses the use of identity management systems for data minimization.
+In these systems usually an identify provider issues credentials that can be used to access a
+service without revealing the user's identity by relying on the authentication assertion from
+the identity provider (see {{Section 6.1.4 of RFC6973}}). This describes a specific form of
+privacy partitioning, simlarly as used for privacy pass (see Section {{privacypass}}).
```

Comment:
I'm not sure this identity provider detail belongs in our intro? I think it is different than the privacy pass approach — the RFC6973 text seems to be more about only collecting what you need for an account, etc.